readthedocs/proxito/views/mixins.py

```diff
@@ -118,6 +118,9 @@ def _serve_401(self, request, project):
         log.debug('Unauthorized access to %s documentation', project.slug)
         return res
 
+    def allowed_user(self, *args, **kwargs):
+        return True
```

Comment:
My idea was to default this to `False` to avoid mistakes if it's not overwritten in the class that inherit from this. Forcing the class that inherit from this to re-define it.

Reply:
Yea, this worried me a bit, we should fix it in another PR.